code/modules/events/_event.dm

```diff
@@ -23,9 +23,6 @@
 	var/alert_observers = TRUE //should we let the ghosts and admins know this event is firing
 									//should be disabled on events that fire a lot
 
-	var/list/gamemode_blacklist = list() // Event won't happen in these gamemodes
```

Comment:
Removing these without replacement doesn't stil well with me but i suppose we're running dynamic long enough that if these were an issue someone would complain.